lib/src/components/navigation bar/navigation_bar.dart

```diff
@@ -56,6 +57,7 @@ class ZetaNavigationBar extends ZetaStatelessWidget {
     this.onTap,
     this.dividerIndex,
     this.semanticLabel,
+    this.shrinkItems = false,
```

Comment:
Can you add shrinkItems into widgetbook please

Reply:
I always forget widgetbook. Thanks for reminding me!